docs/reference/contributing/guidelines/workflow.md

```diff
@@ -79,6 +87,43 @@ Each feature has a tech lead. This person is responsible for:
 - Rebasing often to track master development.
 - Reviewing any addition to the feature branch (approval required by the feature tech lead or another assigned person).
 
+##### Pull request types
+
+We consider the following pull request types.
+
+##### Fix
+
+A bug fix is backward-compatible internal change that fixes incorrect behavior. The fix provides a description what and how it fixes the issue. Tests are included to catch the issue in the future and confirms the correct behavior.
+
+Release: patch
+
+##### Refactor
+
+Refactors are intended for feature releases if they are not fixing specific issue as they 
```

Comment:
Is code cleanup considered a refactor?

Reply:
Also, is the newline between 102 and 103 intended?

Reply:
> Is code cleanup considered a refactor?

It really depends on the scope and changes made.